ecs/cluster.yaml

```diff
@@ -584,6 +584,11 @@ Resources:
                 log_group_name = /var/log/ecs/ecs-init.log
                 log_stream_name = {instance_id}/var/log/ecs/ecs-init.log
                 datetime_format = %Y-%m-%dT%H:%M:%SZ
+              mode: '000644'
+              owner: root
+              group: root
+            '/etc/awslogs/config/ecs.conf':
```

Comment:
What about moving `/var/log/ecs/ecs-init.log` to this section as well?

Reply:
good catch